package.json

```diff
@@ -41,10 +41,11 @@
 		"jest": "^26.0.1",
 		"lint-staged": "^10.0.4",
 		"nock": "^13.0.5",
-		"prettier": "2.0.5",
+		"prettier": "2.6.2",
```

Comment:
this should be done on master, not this PR

Reply:
hah, I actually updated some time ago: https://github.com/siimon/prom-client/pull/510

so this just needs a rebase/merge of `master`